mozilla/foundation.mozilla.org
# removed anchor link for sub-publication pages

Closes #7136 

**Link to sample test page**: https://foundation-s-7136-publi-8sfeai.herokuapp.com/en/publication-page-with-chapter-pages/

**Steps to test**:
1.) Visit https://foundation-s-7136-publi-8sfeai.herokuapp.com/en/publication-page-with-chapter-pages/, and scroll down to view the child publication pages.
2.) Hover over the title of the child publication pages, note that they are no longer clickable as requested.
3) Hover over the 1st chapter of the publication page table of contents, it should be clickable and link correctly.
4.) If everything is working correctly, testing is done! If there are any additional requests, please let me know and I will implement them ASAP. Thank you! 

## Checklist

**Changes in Models:**
